lib/page/process-uploads/process-uploaded-files.js

```diff
@@ -12,7 +12,7 @@ const processUploadedFiles = async (processedResults, uploadControls) => {
     }).map(uploadControl => {
       file.maxSize = uploadControl.maxSize
       file.expires = uploadControl.expires
-      file.allowed_types = uploadControl.allowed_types
+      file.allowed_types = uploadControl.accept
```

Comment:
Never seen variable assignment as a side effect of a .map
Glossing over this for now.